EditorPluginExample/Resources/EditorData/Example.tb.txt

```diff
@@ -0,0 +1,14 @@
+TBLayout: axis: y, distribution: gravity, position: left
+	TBLayout:
+		TBTextField:  id: example_hello, text: "Hello.."
```

Comment:
I keep wanting to fix the TurboBadger UI def file parser to handle spaces and not require tabs every time I see a new tb file, there is an issue